manifesto/personas.md

```diff
@@ -34,30 +34,34 @@ The following personas are fictional characters that we defined in order to repr
+| Characteristics                   | Sebastian wants to bring innovation to products developed at **PureJoy**. He understands the needs of his organisation and seeks for solutions in the CNCF landscape. He evaluates CNCF projects to identify what those tools bring for free and what are the gaps that need to be implemented by the teams in his organisation. He needs to convince his bosses and his development team to new solutions and he is accountable for his recommendations. |
+| Kubernetes experience (1-6)       | :book: :book: :book: :book:                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| Accountability/Decisiveness (1-6) | 💪 💪 💪 💪 💪 💪                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| Slogans                           | _I know what I want... not yet sure how to achieve it._                                                                                                                                                                                                                                                                                                                                                                                                   |
 
 ### GAINS
 
-- :green_heart: I need a comprehensive set of opinionated tools (CNCF-approved, recognized by the community).
-- :green_heart: I need certainty that Kyma uses industry standards.
+- :green_heart: I need to evaluate Kyma fast (within 1 day).
+- :green_heart: I love tutorials, sample code, recepies that refer to my use cases (i.e authentication/authorisation flows).
+- :green_heart: I want to tick off on product (industry) standards for free (i.e security, accessibility, observability, tracing, etc).
+- :green_heart: I want future proof platform - based on references.
+- :green_heart: With Kyma I want to connect to systems I know (public and internal systems).
 - :green_heart: I need a trial version of Kyma before I can recommend it.
-- :green_heart: I don't want to take extra care about security/monitoring/tracing...
-- :green_heart: I need developers in my organization to be focused on the solutions rather than infrastructure.
+- :green_heart: I need agility in architecture designs - I want to replace individual modules if needed.
```

Comment:
```suggestion
- :green_heart: I need agility in architecture design. I want to replace individual modules if needed.
```
I would treat it as an abstract noun here, unless you're talking about a particular design, not the process.